index.html

```diff
@@ -1,15 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Velzck Portfolio V2.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
     <link rel="stylesheet" href="style/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Courier+Prime:wght@400;700&family=Crete+Round&display=swap"
+        rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Courier+Prime:wght@400;700&family=Crete+Round&family=Poppins&display=swap"
+        rel="stylesheet">
     <title>Portfolio </title>
 </head>
+
 <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+    <header>
+
+        <div id="header" class="flexColumn">
+            <div id="headerNav" class="headerCont roboto">
+                <p id="hour">VelzckC0D3</p>
+                <img id="menu" src="style/img/menu.svg" alt="">
```

Comment:
Please consider adding a  `nav` tag to your HAMBURGER icon.
    - The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.

Reply:
Done, thank you for your correction! appreciated

Reply:
Done, thank you for your correction! appreciated
![image](https://user-images.githubusercontent.com/92229666/219451525-2363d2fc-4beb-4e2b-9943-9b8c7825f4c8.png)